packages/address/src/tests/utilities.test.ts

```diff
@@ -0,0 +1,5 @@
+import {renderLineTemplate} from '../utilities';
+
+describe('renderLineTemplate()', () => {
+  it('TODO', () => {});
```

Comment:
I'd rather see a list of what you plan to test via `it.skip`, then we can decide if it makes sense to write the tests in this PR or defer it for another one. We're trying not to include any untested library code though.

Reply:
Oh shoe, I'm sorry, I completely missed that one... 🤦‍♂️